pilz_psenscan_test_area/setup_workspace.sh

```diff
@@ -3,6 +3,7 @@
 create_catkin_ws()
 {
   mkdir -p "$SRC_DIR" 2>&1
+  catkin init --workspace "$CATKIN_WS_DIR"
```

Comment:
```suggestion
  catkin clean --workspace "$CATKIN_WS_DIR" --deinit -y
  catkin init --workspace "$CATKIN_WS_DIR"
```

Reply:
Maybe just adding the `--reset` flag to `catkin init` accomplishes the same.

Reply:
Is there any harm in not cleaning the workspace?

Reply:
never mind